server/model/yaml/encoding.go

```diff
@@ -58,6 +59,13 @@ func Decode(contents []byte) (File, error) {
 			return File{}, fmt.Errorf("cannot decode config: %w", err)
 		}
 		f.Spec = config
+	case FileTypePollingProfile:
```

Comment:
Is this consumed somewhere other than the CLI? If so, let's remove this and try to keep the new things decoupled from this old approach